src/QirRuntime/build.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# DEPRECATED !!!
+
```

Comment:
I'd suggest replacing this with a Python-style docstring:

```suggestion
"""
WARNING: Note that this script has been deprecated.
          Please use the `build-qir-runtime.ps1` script instead.
          See README.md for more details.
"""
```